app/views/coronavirus_form/check_your_answers/helpers/_summary_list_multiple_items.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="gem-c-summary-list ">
+  <h2 class="govuk-heading-l gem-c-summary-list__group-title"><%= question_title %></h2>
+  <ul class="govuk-summary-list__actions-list gem-c-summary-list__group-actions-list">
+    <li class="govuk-summary-list__actions-list-item"> <a class="govuk-link" href="<%= question_href %>?change-answer">
+    Change<span class="govuk-visually-hidden"></span></a></li>
```

Comment:
This `span` isn't doing anything - so we can either remove it or add the question in here so it's read aloud by a screen reader.

Reply:
Thanks @injms 👍

Reply:
This is outdated 😺